public/app/features/auth-config/fields.tsx

```diff
@@ -655,25 +805,33 @@ export function fieldMap(provider: string): Record<string, FieldData> {
                 }
                 return true;
               },
-              message: 'Team Ids must be numbers.',
+              message: t('auth-config.fields.team-ids-numbers', 'Team IDs must be numbers.'),
             }
           : undefined,
     },
     hostedDomain: {
-      label: 'Hosted domain',
-      description: 'The domain under which Grafana is hosted and accessible.',
+      label: t('auth-config.fields.hosted-domain-label', 'Hosted domain'),
+      description: t(
+        'auth-config.fields.hosted-domain-description',
+        'The domain under which Grafana is hosted and accessible.'
+      ),
       type: 'text',
     },
     validateHd: {
-      label: 'Validate hosted domain',
-      description:
-        'If enabled, Grafana will match the Hosted Domain retrieved from the Google ID Token against the Allowed Domains list specified by the user.',
+      label: t('auth-config.fields.validate-hosted-domain-label', 'Validate hosted domain'),
+      description: t(
+        'auth-config.fields.validate-hosted-domain-description',
+        'If enabled, Grafana will match the Hosted Domain retrieved from the Google ID Token against the Allowed Domains list specified by the user.'
```

Comment:
Could you please extract the "Allowed Domains" part into a label?
```suggestion
        'If enabled, Grafana will match the Hosted Domain retrieved from the Google ID Token against the "{{ allowedDomainsLabel }}" list specified by the user.'
```